open-hackathon-server/src/hackathon/hazure/file_service_adapter.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+"""
+Copyright (c) Microsoft Open Technologies (Shanghai) Co. Ltd.  All rights reserved.
+
+The MIT License (MIT)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+"""
+
+__author__ = "rapidhere"
+__all__ = ["FileServiceAdapter"]
+
+from azure.storage.file import FileService
+
+from service_adapter import ServiceAdapter
+
+
+class FileServiceAdapter(ServiceAdapter):
+    """This adapater is a part of AzureStorage Python SDK adapter
+
+    this is just a thin proxy on azure.storage.file.FileService, and have the same usage as BlobServiceAdapter
```

Comment:
Sounds like `FileServiceAdapter` is duplicated with `FileServiceAdapter`? If it is, please remove one of them